controllers/helmchartproxy/helmchartproxy_controller_phases_test.go

```diff
@@ -191,6 +191,8 @@ var (
 )
 
 func TestReconcileForCluster(t *testing.T) {
+	t.Parallel()
```

Comment:
We're already calling this in line 274 in the loop iteration. I'm not sure if we need to call it at the beginning here.

Reply:
Yes, it seems like we have to do it... The error message looks like: `TestReconcileDelete should call t.Parallel on the top level as well as its subtests (tparallel)`